tests/integration/helpers/selectLocale.js

```diff
@@ -0,0 +1,11 @@
+import { t, Selector } from "testcafe";
```

Comment:
can we import `t` as a meaningful name?
something like?
```
import { t as testcafe, Selector} from "testcafe"
```

Reply:
Yes, that's better. I agree. Will make this change.